wiki/elf-to-mdx.mjs

```diff
@@ -48,11 +48,10 @@ export function elfToMdx(elfFilename, elfFile) {
       body.push(
         "",
         "<Twelf " +
-          (checked
-            ? "checked context={`" +
-              escapeBacktickEnv(twelfcontext.join("\n")) +
-              "`} "
-            : "") +
+          (checked ? "checked " : "") +
+          "context={`" +
+          escapeBacktickEnv(twelfcontext.join("\n")) +
+          "`} " +
```

Comment:
The converter now _always_ adds the context, since that's relevant to loading a non-checked-decl snippet into Twelf Live.

Reply:
I'm open-minded about in the future restoring a proper context-and-incremental-snippet feature flow into twelf live experience

Reply:
yep, that can is happy to get kicked down the road